Makefile.am

```diff
@@ -56,7 +56,7 @@ TAR_OPTIONS = --owner=0 --group=0
 export TAR_OPTIONS
 
 
-EXTRA_DIST = ChangeLog INSTALL README.md LICENSE CFVERSION
+EXTRA_DIST = ChangeLog INSTALL README.md LICENSE CFVERSION docs/custom_promise_types/cfengine.sh
```

Comment:
Once again I have to say that these things shouldn't be under *docs/*. What about *misc/custom_promise_types*?

Reply:
And then this should be added to `EXTRA_DIST` in the particular folder, not in the top-level *Makefile.am*.

Reply:
I don't have a strong opinion on this, but what would the person who put them there, in the first place, say? @olehermanse ?

Reply:
They are in `docs/custom_promise_types/` just to have everything in 1 place. I think the libraries should be moved to `masterfiles/modules/`